src/tss2-esys/esys_int.h

```diff
@@ -54,7 +54,6 @@ typedef struct {
     TPM2B_NONCE *nonceCaller;
     TPM2B_NONCE nonceCallerData;
     TPM2B_ENCRYPTED_SECRET *encryptedSalt;
```

Comment:
This is also unused. probably easiest to remove them both in 1 commit ?

Reply:
Done